lib/public/IURLGenerator.php

```diff
@@ -49,7 +49,7 @@ interface IURLGenerator {
 	 *
 	 * This is a copy of the frontend regex in core/src/OCP/comments.js, make sure to adjust both when changing
 	 *
-	 * @since 25.0.0
+	 * @since 25.0.0 changed to match localhost and hostnames with ports in Nextcloud 29
```

Comment:
```suggestion
	 * @since 25.0.0
	 * @since 29.0.0 changed to match localhost and hostnames with ports
```

And also on `URL_REGEX` please :)

Reply:
Alright, added now :blush: